packages/palette/src/elements/Radio/Radio.tsx

```diff
@@ -145,13 +147,14 @@ const hoverStyles = ({ selected, hover }) => {
   }
 }
 
-interface ContainerProps extends FlexProps {
+interface ContainerProps extends ClickableProps {
   disabled: boolean
   hover: boolean
   selected: boolean
 }
 
-const Container = styled(Flex)<ContainerProps>`
+const Container = styled(Clickable)<ContainerProps>`
```

Comment:
Actually, I need to double check this; probably shouldn't be a real button since the label is supposed to be triggering the hidden input.